code/tier_classification/tier_classification.R

```diff
@@ -90,18 +95,12 @@ if(nrow(oncokb_anno) == 0){
     dplyr::filter(gnomAD_AF<0.01| is.na(gnomAD_AF)) %>%
     # filter on VAF over 5% to avoid background noise
     dplyr::filter(VAF>0.05) %>%
-    # filter to contain only mutations that has COSMIC ID
-    dplyr::mutate(has_COSMIC = dplyr::case_when(
-      stringr::str_detect(Existing_variation,
-                          "COSM") ~ "Yes",
-      TRUE ~ "No"
-    )) %>%
-    dplyr::filter(has_COSMIC == "Yes") %>%
+    # filter to contain only mutations that has COSMIC ID or has resistance mutation
+    dplyr::filter(grepl("COSM", Existing_variation) | cosmic_resistance_variant == "Yes") %>% 
```

Comment:
Same thing here for Tier II- eliminating from this section and instead editing line 93 to look in cosmic resistance file for a variant whose 'Tier' field is left blank (so a non-tier 1 resistance variant.